docs/python/tutorial-azure-functions.md

```diff
@@ -236,18 +236,43 @@ In these steps, you use the Functions extension to create a "Function App" on Az
     - In that resource group, create the storage account, App Service Plan, and App Service to host your code.
     - Deploy your code to the Function app.
 
-    You can also see progress in the **Azure: Functions** explorer:
+    The **Azure: Functions** explorer also shows progress:
 
     ![Deployment progress indicator in the Azure: Functions explorer](images/functions/deploy-progress.png)
 
-1. Once deployment is complete, the **Output** window shows the public endpoint on Azure:
+1. Once deployment is complete, the Azure Functions extension displays a message with buttons for three additional actions:
+
+    ![Message indicating successful deployment with additional actions](images/functions/deployment-popup.png)
+
+    For **Stream logs** and **Upload settings**, see the next sections. For **View output**, see step 5 that follows.
+
+1. After deployment, the **Output** window also shows the public endpoint on Azure:
 
     ```
     HTTP Trigger Urls:
       HttpExample: https://vscode-azure-functions.azurewebsites.net/api/HttpExample
     ```
 
-1. Use this endpoint to run the same tests you did locally, using URL parameters and/or requests with JSON data in the request body. You should see the same results from the public endpoint as you did locally.
+    Use this endpoint to run the same tests you did locally, using URL parameters and/or requests with JSON data in the request body. The results of the public endpoint should match those when you ran the function locally.
+
+### Stream logs
+
+Support for log streaming is currently in development, as described on [Issue 589](https://github.com/microsoft/vscode-azurefunctions/issues/589) for the Azure Functions extension. The **Stream logs** button in the deployment message popup will eventually connect the log output on Azure to VS Code. You will also be able to start and stop the log stream on the **Azure Functions** explorer by right-clicking the Functions project and selecting **Start streaming logs** or **Stop streaming logs**.
+
+At present, however, these commands aren't yet operational. Log streaming is instead available in a browser by running the following command, replacing `<app_name>` with the name of your Functions app on Azure:
+
+```bash
+# Replace <app_name> with the name of your Functions app on Azure
+func azure functionapp logstream <app_name> --browser
+```
+
```

Comment:
@asavaritayal Can you review the Stream logs section again with these revisions.

Reply:
Looks good, thanks @kraigb !